CHANGELOG.md

```diff
@@ -41,12 +41,14 @@
 * Added `conda-forge::pygments=2.5.2`
 * Removed `conda-forge::r-markdown=1.0`
 
-## v1.0.0 - 2019-09-19
+## [v1.0.0] - 2019-09-19
```

Comment:
Whilst valid, I prefer inline links for nearly all use cases in Markdown. It's the style that we tend to use with nf-core and personally I find it easier to keep track of / maintain.

```suggestion
## [v1.0.0](https://github.com/nf-core/smrnaseq/releases/tag/1.0.0) - 2019-09-19
```

Reply:
I agree, that looks cleaner!